semeio/jobs/scripts/csv_export2.py

```diff
@@ -0,0 +1,37 @@
+import os
+import argparse
+
+import pandas as pd
+
+from res.enkf import ErtScript
+from fmu import ensemble
+
+
+class CsvExport2Job(ErtScript):
+    def run(self, *args):
```

Comment:
Can  we pull this run() out of the class, so that we can also  facilitate a command line endpoint doing the same with the same arguments?

Reply:
ErtScript requires a run() method in the class inherit from it. Do you want an additional command line endpoint? And if so, where?

Reply:
But since the `run` function is not using `self` at all I'm guessing this does not even need to be an internal script ;) Which makes sense as it crawls the file system ;)

Reply:
I think I want an additional endpoint also yes, very useful for debugging, and for relaunching exports without interferring with a config file.

Reply:
ok, i'll move it!

Reply:
Are we sure we should expose another endpoint? With future design in mind an export from ERT should be done with a configuration file/experiment in mind and not an ensemble on disk. Otherwise we immediately have to break this behaviour again when we fetch the data from the API.

Which use case is it that we are not covering by having the workflow `CSV_EXPORT2` available and supporting to run it via the ERT cli as: `ert workflow CSV_EXPORT2 my_config.ert`?

Reply:
The endpoint may belong to fmu-ensemble (it has been planned for years but I have not been able to come up with a name, or how the command line api should look like). The command line api is simpler for this semeio job, since the difficult part is put into the runpath file.

The use-case for an endpoint is being able to do reliable (test/debugging) exports, perhaps on other users data,  without relying on whether the config file has been tampered with, or knowing which config file to pick. Or if you generate your own runpath file. Not  sure how relevant it is for regular users, so perhaps the endpoint should be somewhat hidden.

Reply:
That makes sense, but I think the data that ERT serves should be based around configuration files/experiments. And this is why we need to do persist configuration files after they have been ran to be able to serve this even more reliably in the future.

I would suggest putting that second end point in `fmu-ensembles` or another repository.

For ERT I'm thinking that we should be able to do `ert cases` to get a list of the cases available and then `ert export <case_name>` to get an export of the data from that case instead of basing it on the scratch-disk. This should still fly as long as you have access to storage, but will shift the perceived location of master data from scratch to storage. Which makes 100% sense given that storage is a place you can trust with your data ;)